t/ids_from_pages.rb

```diff
@@ -0,0 +1,30 @@
+require 'test_helper'
+
+describe 'Wikidata#ids_from_pages' do
+  around { |test| VCR.use_cassette(titles.join('__and__'), &test) }
```

Comment:
I'm not 100% convinced by this approach (particularly if we want to add a test case for a version that fetches a lot of data), but we can live with it for now, and fix it later if/when needed.